Eshop/apps/orders/models.py

```diff
@@ -1,3 +1,32 @@
 from django.db import models
+from apps.products.models import Product
 
-# Create your models here.
+class Order(models.Model):
+    order_number = models.PositiveSmallIntegerField(default=1, unique=True)
```

Comment:
```suggestion
    order_number = models.PositiveSmallIntegerField(default=randint, unique=True)
```

https://docs.djangoproject.com/en/2.2/ref/models/fields/#uuidfield uuid4